docs/templates.md

```diff
@@ -80,6 +80,12 @@ This type cannot be printed. The following methods are defined.
   working-copy commit as `<workspace name>@`.
 * `current_working_copy() -> Boolean`: True for the working-copy commit of the
   current workspace.
+* `bookmarks() -> List<RefName>`: Local and remote bookmarks pointing to the 
+  commit. A tracking remote branch will be included only if its target is 
+  different from the local one.
+* `local_bookmarks() -> List<RefName>`: All local bookmarks pointing to the commit.
+* `remote_bookmarks() -> List<RefName>`: All remote bookmarks pointing to the commit.
+<!-- TODO: Remove in jj 0.28+ -->
```

Comment:
nit: I think deprecated names can be just removed from the docs

Reply:
done